### ISSUE TITLE
Fimir improvements

### ISSUE THREAD

**Issue by Rudolfssis** [Hexenstag-Team/WH-Hexenstag-Dev#69]
**Describe the feature/solution you'd like**
I would like the **Fimir** to be able to marry and for them to be able to have access to the offmap chaos gods. 
As of now, they are not that fun to play, because of their inability to give out titles to other fimir (because of said inability to marry, despite the ability to take concubines) and because there is nothing else to do once you've become a chaos champion.
In short, can they please be tweaked in order to be more like the beastmen or other chaos rulers?
